lib/vector-core/src/event/lua/value.rs

```diff
@@ -53,11 +53,15 @@ mod test {
     use chrono::{TimeZone, Utc};
 
     #[test]
+    #[allow(clippy::approx_constant)]
```

Comment:
Could we add a comment for this and all other `allow` about why we allow them? I think that would help when revisiting them later.

Reply:
My appetite for commenting all of them is pretty low, but that's surely a function of how I deal with lints like this. That is, if I'm curious why it's in place I'll flag it off and let clippy explain what's up. That said, I don't want to make the code base beholden to my idiosyncratic way of working with lints. If there are any that seem particularly confounding to you -- the quickcheck ones seem like a candidate --  I'll gladly give them a comment.

Reply:
👍 that tells you what it is complaining about, but it is another step to why we are ok with it; which may not always be apparent. I'm more concerned about documenting why we disable it than what specifically it is complaining about -- albeit they are related.

Reply:
Yeah, that makes good sense to me. Will do.